openc3/lib/openc3/microservices/interface_microservice.rb

```diff
@@ -164,7 +171,7 @@ def run
                 if target_name
                   command = System.commands.identify(cmd_buffer, [target_name])
                 else
-                  command = System.commands.identify(cmd_buffer, @cmd_target_names)
+                  command = System.commands.identify(cmd_buffer, @interface.cmd_target_names)
```

Comment:
This is just a straight up bug. How have we never hit this before? And therefore is it even necessary?

Reply:
This code is needed if not using the PREIDENTIFIED interface